robber/expect.py

```diff
@@ -19,8 +19,9 @@ class expect:
     matchers = {}
     message = None
 
-    def __init__(self, object):
-        self.object = object
+    def __init__(self, obj):
+        self.obj = obj
+        self.is_negative = False
```

Comment:
@hieueastagile Could you explain why? As I mentioned in this commit https://github.com/EastAgile/robber.py/pull/15/commits/de70d8cf67872c0edd981f803d0f4deb12129933 

> Base on OALD, "negative" means "containing a word such as 'no', 'not',
'never', etc.". Moreover, I cannot find the word "negated".